.github/workflows/release-maven-perform.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Run Maven Release Perform.
```

Comment:
`Maven Release Perform` => `Maven Perform`

Reply:
Release in middle in name of plugin.

It is no maven activity, release can be done in many ways, we use this simple general plugin.

Reply:
@rnveach , 
What about this 
`Run Perform goal of Maven Release plugin .`?

Reply:
Done.